x-pack/plugins/cases/kibana.json

```diff
@@ -1,7 +1,7 @@
 {
   "configPath":[
-     "cases",
-     "xpack"
+     "xpack",
```

Comment:
nit - can we avoid these changes ;)

Reply:
I don't think so, because `/cases/xpack` is incorrect path ;)